src/site/_includes/pattern-set.njk

```diff
@@ -122,6 +123,20 @@ pageScripts:
         {% endif %}
       {% endfor %}
 
+      {# Patterns that belong directly to the pattern suite #}
+      {% for patternId, pattern in patterns.patterns %}
+        {% if pattern.set == patternSuite.id %}
+          {% if loop.first %}
+            <h2> Other patterns</h2>
```

Comment:
You should make this an i18n string. I don't think there's any other literal English on this page.